test/metadata_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+require 'ttnt/metadata'
+
+class MetaDataTest < TTNT::TestCase
+  def setup
+    @storage_file = "#{@repo.workdir}/.ttnt"
+    FileUtils.rm_rf(@storage_file)
```

Comment:
That's true, thank you! Fixed: https://github.com/Genki-S/ttnt/commit/1ec9320e6d6dff2aaa6e1d42fe42807477255e2c